csrankings.csv

```diff
@@ -11115,7 +11115,7 @@ Miguel Leitão Bignolas Mira da Silva,Universidade de Lisboa,https://fenix.tecni
 Miguel M. Silva,Universidade de Lisboa,https://fenix.tecnico.ulisboa.pt/homepage/ist13948,8Lmbw5kAAAAJ
 Miguel Matos,Universidade de Lisboa,http://haslab.uminho.pt/mmatos,OcBYw6kAAAAJ
 Miguel Mira da Silva,Universidade de Lisboa,https://fenix.tecnico.ulisboa.pt/homepage/ist13948,8Lmbw5kAAAAJ
-Miguel Nacenta,University of St Andrews,https://sachi.cs.st-andrews.ac.uk/people/faculty/miguel-nacenta,M_Ne0psAAAAJ
+Miguel A. Nacenta,University of Victoria,https://nacenta.com,M_Ne0psAAAAJ

```

Comment:
Home page says nothing about Victoria (see https://nacenta.com/about/).

Reply:
True! He's listed on [the uvic faculty directory](https://www.uvic.ca/engineering/computerscience/people/faculty/index.php), but hasn't yet updated online presence (webpage, scholar, twitter).

Reply:
OK, assume we aren't forcibly changing his affiliation :). Please ask him to update his web page.